pkg/restore/restorers/service_restorer.go

```diff
@@ -45,7 +47,9 @@ func (sr *serviceRestorer) Prepare(obj runtime.Unstructured, restore *api.Restor
 		return nil, nil, err
 	}
 
-	delete(spec, "clusterIP")
+	if fmt.Sprintf("%v", spec["clusterIP"]) != "None" {
```

Comment:
I think I'd rather do `collections.GetString` perhaps... wdyt @skriss ?

Reply:
@ncdc, Aha, I hadn't seen this function. Certainly not opposed to it, myself.

Reply:
@ncdc I had been thinking the same.